examples/metalk8s-solution-example/ui/src/containers/Layout.js

```diff
@@ -101,20 +101,35 @@ const Layout = props => {
           <PrivateRoute exact path="/about" component={Welcome} />
           <PrivateRoute
             exact
-            path="/customResource"
-            component={CustomResource}
+            path="/environments/:name/clockServer/create"
```

Comment:
I think we should keep case-insensitive URLs:

```suggestion
            path="/environments/:name/clock-servers/create"
```